valhalla/baldr/sign.h

```diff
@@ -26,7 +26,7 @@ class Sign {
    * @param  text_offset  Offset to text in the names/text table.
    */
   Sign(const uint32_t idx, const Sign::Type& type, const bool rn, const uint32_t text_offset)
-      : edgeindex_(idx), type_(static_cast<uint32_t>(type)), is_route_num_(rn), spare_(0),
+      : edgeindex_(idx), type_(static_cast<uint32_t>(type)), is_route_num_(rn), tagged_(0),
```

Comment:
did you still want to zero out spare?

Reply:
oh - ignore, i was looking at wrong spare count

Reply:
no more spare!